website/docs/contributing.md

```diff
@@ -0,0 +1,35 @@
+---
+sidebar_position: 3
+---
+
+# How to contribute to FerretDB
+
+FerretDB is an open source project and everyone – developers and non-developers – is welcome to contribute.
+If you’re interested in contributing to FerretDB projects, this documentation will help you get started.
+
+Here are some of the current FerretDB projects that you can fork and contribute to:
+
+* [FerretDB](https://github.com/FerretDB/FerretDB): This repository contains the FerretDB project – an open source alternative project to MongoDB.
+* [github-actions](https://github.com/FerretDB/github-actions): This repository houses our shared GitHub Actions for FerretDB, dance, and other repositories.
+* [dance (stub)](https://github.com/FerretDB/dance): This repository is a FerretDB integration testing tool.
```

Comment:
Dance is not a stub :) Not sure why I wrote that.

Also, I think we should move it above github-actions

Reply:
That wasn't done